e2e/features/image-download/unsupported-test.js

```diff
@@ -6,7 +6,7 @@ const startParams = ['v=-180,-90,180,90', 't=2018-06-01', 'imageDownload='];
 const TIME_WAIT = 10000;
 
 module.exports = {
-  after: function(client) {
+  after(client) {
```

Comment:
@jasontk19 why does this differ from `before: (c) => {` change below? I don't know what rule this is implementing.  I would expect `after: (client) =>`

Reply:
I'm not sure, most of this kind of formatting was handled by the autofix.  This is a change to comply with the `object-shorthand` rule for methods: https://github.com/airbnb/javascript#es6-object-shorthand

I'm not sure why it wouldn't also format the other to `before(c) { ... ` 🤷‍♂